src/main/resources/jupyter/init-actions.sh

```diff
@@ -302,7 +303,9 @@ if [[ "${ROLE}" == 'Master' ]]; then
         JUPYTER_USER_SCRIPT=`basename ${JUPYTER_USER_SCRIPT_URI}`
         docker cp /etc/${JUPYTER_USER_SCRIPT} ${JUPYTER_SERVER_NAME}:${JUPYTER_HOME}/${JUPYTER_USER_SCRIPT}
         retry 3 docker exec -u root ${JUPYTER_SERVER_NAME} chmod +x ${JUPYTER_HOME}/${JUPYTER_USER_SCRIPT}
-        retry 3 docker exec -u root -e PIP_USER=false ${JUPYTER_SERVER_NAME} ${JUPYTER_HOME}/${JUPYTER_USER_SCRIPT}
+        docker exec -u root -e PIP_USER=false ${JUPYTER_SERVER_NAME} ${JUPYTER_HOME}/${JUPYTER_USER_SCRIPT} &> us_output.txt || EXIT_CODE=$? && true ;
```

Comment:
so we're no longer retrying?

Reply:
Yes. We previously added retries to all docker exec statements because some automation tests were failing transiently. Rob and I talked about it for a bit and it seems like it would make most sense to keep the retries everywhere except here because in the case of failing userscript that takes a lot of time to run, the user would have to wait three times the length of the runtime to get an error message

Reply:
Does `&>` make it run in the background? I think the init script should block on the execution of the user script.

Reply:
Oh nevermind -- `&>` means `>us_output 2>&1`. This seems reasonable then I think..